src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -25,11 +26,10 @@ public class ArmSubsystem extends SubsystemBase{
     private final double softStop = 20.0;
     private final double hardStop = 25.0;
 
-    private CANSparkMax leftIntakeMotor;
-    private CANSparkMax rightIntakeMotor;
+    private IntakeSubsystem intake;
 
     private double shooterGoalPower;
-    //private double intakeGoalPower;
+    private double intakeGoalPower;
```

Comment:
Is this useful anymore now that all intake things are in the new intake class?

Reply:
deleted that and shooterGoalPower (was never used)